.travis/test.sh

```diff
@@ -24,6 +24,12 @@ then
   UT_FLAGS+=" -K FIXME_py3"
 fi
 
+if [[ $TOXENV == py37* ]]
```

Comment:
You should test for `TRAVIS_DIST=xenial` and open an issue to track this regression.

Reply:
https://github.com/secdev/scapy/issues/1696